pkg/gatherers/conditional/gathering_functions.go

```diff
@@ -13,13 +13,19 @@ type GatheringFunctions = map[GatheringFunctionName]interface{}
 // GatheringFunctionName defines functions of conditional gatherer
 type GatheringFunctionName string
 
-// GatherLogsOfNamespace is a function collecting logs of the provided namespace.
-// See file gather_logs_of_namespace.go
-const GatherLogsOfNamespace GatheringFunctionName = "logs_of_namespace"
+const (
+	// GatherLogsOfNamespace is a function collecting logs of the provided namespace.
+	// See file gather_logs_of_namespace.go
+	GatherLogsOfNamespace GatheringFunctionName = "logs_of_namespace"
 
-// GatherImageStreamsOfNamespace is a function collecting image streams of the provided namespace.
-// See file gather_image_streams_of_namespace.go
-const GatherImageStreamsOfNamespace GatheringFunctionName = "image_streams_of_namespace"
+	// GatherImageStreamsOfNamespace is a function collecting image streams of the provided namespace.
+	// See file gather_image_streams_of_namespace.go
+	GatherImageStreamsOfNamespace GatheringFunctionName = "image_streams_of_namespace"
+
+	// GatherAPIRequestCounts is a function collecting api request counts for the resources read
+	// from the corresponding alert
+	GatherAPIRequestCounts GatheringFunctionName = "api_request_counts"
```

Comment:
Maybe we could call it api_request_counts_of_alert or something similar to be more specific?